### PR TITLE
[branch/v8] update doc examples to change from admin role to editor,access

### PR DESCRIPTION
**Pull request by stevenGravy**
backport #9334

### CODE REVIEW

**Review by klizhentas — APPROVED** [review]

File: docs/pages/setup/deployments/aws-terraform.mdx [hunk 1]
```diff
@@ -467,7 +467,7 @@ $ ssh -i ${TF_VAR_key_name}.pem -o ProxyCommand="ssh -i ${TF_VAR_key_name}.pem -
 
 ```code
 # From EC2 host
-$ sudo tctl users add teleport-admin --roles=admin
+$ sudo tctl users add teleport-admin --roles=editor,access --roles=root
```
Comment (ptgott) [inline docs/pages/setup/deployments/aws-terraform.mdx:470]: Should the second `--roles` flag in this command be `--logins`?

**Review by zmb3 — CHANGES_REQUESTED** [review]
This is not identical to the commit being backported. See @ptgott's comment on the `--logins` flag.

**Comment by klizhentas** [comment]
@stevenGravy please address Pauls' comments and merge

File: docs/pages/setup/deployments/aws-terraform.mdx (hunk 1 above)
Comment (stevenGravy) [reply docs/pages/setup/deployments/aws-terraform.mdx]: fixed

**Review by zmb3 — APPROVED** [review]

**Pull request merged**